Dockerfile.helm3

```diff
@@ -2,6 +2,8 @@ FROM golang:1.13.7-alpine3.11 as builder
 
 RUN apk add --no-cache make git
 WORKDIR /workspace/helmfile
+COPY go.mod go.sum ./
+RUN go mod download
```

Comment:
The point of `vendor` was to avoid running `go mod download` (or plain `go build` that results in downloading deps not in the cache) every time we run a docker build.

Reply:
thanks for the reply, close it